test/rules/prefer_contains_test.dart

```diff
@@ -27,6 +27,7 @@ condition() {
 ''', [
       // No lint
       error(HintCode.UNUSED_LOCAL_VARIABLE, 41, 4),
+      error(CompileTimeErrorCode.ARGUMENT_TYPE_NOT_ASSIGNABLE, 77, 3),
```

Comment:
I don't understand where this is coming from. I think the code is valid... `var next;` makes a dynamic, so the assignment should be valid.

Reply:
The issue is that the list is of type `int` and we're passing in a String to `indexOf`.

<img width="1162" alt="image" src="https://user-images.githubusercontent.com/67586/181078599-fcb974e7-bcd6-4f8d-bad6-d38d84f48797.png">

Reply:
Ah I see. So strange that this wasn't picked up before...

Reply:
Yeah.  That's why I bumped the analyzer lower bound.  I'm guessing this got fixed in `4.3.1`.  (@scheglov?)

Reply:
I don't remember fixing anything related recently.